package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
Things like `baseUrl` and `{{assetsUrl}}` do keep the app from running via `ember serve` or from being able to run the tests from `/tests` and they were not deprecations but actual changes.

I guess it depends from which lens you choose to view things.  In my opinion part of being an Ember add-on is for these things to be functional.  For them to break means the add-on is broken.  Sure, it's not from a consumers perspective but it is for any development work, which is how I discovered it and had to perform the init for multiple versions.